apertium_apy/apy.py

```diff
@@ -182,7 +183,8 @@ def parse_args(cli_args=sys.argv[1:]):
     parser.add_argument('pairs_path', help='path to Apertium installed pairs (all modes files in this path are included)')
     parser.add_argument('-s', '--nonpairs-path', help='path to Apertium tree (only non-translator debug modes are included from this path)')
     parser.add_argument('-l', '--lang-names',
-                        help='path to localised language names sqlite database (default = langNames.db)', default='langNames.db')
+                        help='path to localised language names sqlite database (default = langNames.db)',
+                        default=pkg_resources.resource_filename('tempfile','langNames.db')
```

Comment:
What does the `tempfile` do here?

Reply:
tempfile.py is a default python file that generates temporary files. Actually pkg_resources require a module in first arguement and the directory or file in second arguement, so I gave tempfile as reference but I am not sure what location I need to find.